src/wp-content/themes/twentyeleven/inc/theme-options.php

```diff
@@ -107,6 +107,11 @@ function twentyeleven_theme_options_add_page() {
 }
 add_action( 'admin_menu', 'twentyeleven_theme_options_add_page' );
 
+/**
+ * Adds help documentation to the theme options page.
+ *
+ * @since Twenty Eleven 1.0
```

Comment:
```suggestion
 * @since Twenty Eleven 1.3
```

r19237 added `twentyeleven_theme_options_help()` for Twenty Eleven 1.3.